src/main/java/com/greenfox/lvlup/controller/UserBadgeDTO.java

```diff
@@ -0,0 +1,32 @@
+package com.greenfox.lvlup.controller;
```

Comment:
DTOs could be under the model package in an own dto package

Reply:
okay, we moved them there!